specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -1020,15 +1020,15 @@
             "type": "object",
             "description": "describes properties of an auto provisioning setting",
             "properties": {
-                "provisioningSetting": {
+                "autoProvision": {
                     "type": "string",
                     "description": "Describes what kind of security agent provisioning action to take",
-                    "enum": ["Automatic", "Off"],
+                    "enum": ["On", "Off"],
                     "x-ms-enum": {
-                        "name": "provisioningSetting",

```

Comment:
Please acknowledge that this will be a breaking change in the SDKs generated

Reply:
we acknowledge